### PR TITLE
Read configDir and use for `main.js` location

### PR DESCRIPTION
**Pull request by tmeasday**
To QA:

1. `cd src`
2. Remove existing config from `main.js`
3. `CHROMATIC_ADDON_NAME='../dist/index.js' ../node_modules/.bin/storybook dev -c ../.storybook -p 6006`
4. Select a project, check `main.js` has id/token

<details>
  <summary>📦 Published PR as canary version: <code>0.0.25--canary.39.7d57e7f.0</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @chromaui/addon-visual-tests@0.0.25--canary.39.7d57e7f.0
  # or 
  yarn add @chromaui/addon-visual-tests@0.0.25--canary.39.7d57e7f.0
  ```
</details>

### CODE REVIEW

**Review by ghengeveld — APPROVED** [review]
What should we do about the hardcoded `main.js` in the addon UI?

File: src/index.ts [hunk 1]
```diff
@@ -81,7 +81,7 @@ async function serverChannel(
     async ({ projectId, projectToken: updatedProjectToken }: UpdateProjectPayload) => {
       projectToken = updatedProjectToken;
 
-      const mainPath = await findConfig("main");
```
[the remark this answers is not in the document]
Comment (tmeasday) [reply src/index.ts:84]: In https://github.com/chromaui/addon-visual-tests/pull/41 I send the `configDir` up on the failed event. I could probably send the main filename also (although we probably need to consider the case that `findConfig` itself fails / errors). And we could send that info on success too WDYT?

[the remark this answers is not in the document]
Comment (ghengeveld) [reply src/index.ts:84]: Yes that seems appropriate. The alternative would be to not show the filename in the UI (i.e. be vague about it) but that'll likely come back to bite us later.

[the remark this answers is not in the document]
Comment (tmeasday) [reply src/index.ts:84]: That's what i did the failure case. I'll make a ticket to improve.

**Pull request merged**